src/main.rs

```diff
@@ -49,139 +49,198 @@ fn run() -> Result<(), failure::Error> {
                         )
                         .arg(
                             Arg::with_name("title")
+                            .help("New title for the namespace")
                             .required(true)
                         )
                 )
                 .subcommand(
                     SubCommand::with_name("list")
+                        .about("List all namespaces on your Cloudflare account")
                 )
+        )
+            .subcommand(SubCommand::with_name("kv:key")
+                .about(&*format!(
+                    "{} Interact with your Workers KV Key-Value Pairs",
+                    emoji::KV
+                ))
                 .subcommand(
-                    SubCommand::with_name("read-key")
-                        .arg(
-                            Arg::with_name("namespace-id")
-                            .required(true)
-                        )
-                        .arg(
-                            Arg::with_name("key")
-                            .required(true)
-                        )
-                )
-                .subcommand(
-                    SubCommand::with_name("write-key")
+                    SubCommand::with_name("put")
+                        .about("Put a key-value pair into a namespace")
                         .arg(
                             Arg::with_name("namespace-id")
+                            .help("The ID of the namespace this action applies to")
                             .required(true)
+                            // .short("n")
+                            // .long("namespace-id")
+                            // .value_name("<ID>")
+                            // .takes_value(true)
                         )
                         .arg(
                             Arg::with_name("key")
+                            .help("Key to write value to")
                             .required(true)
                         )
                         .arg(
                             Arg::with_name("value")
+                            .help("Value for key")
                             .required(true)
                         )
                         .arg(
                             Arg::with_name("expiration")
+                            .help("Number of seconds since the UNIX epoch, indicating when the key-value pair should expire")
                             .short("e")
                             .long("expiration")
                             .takes_value(true)
                             .value_name("SECONDS")
-                            .help("the time, measured in number of seconds since the UNIX epoch, at which the entries should expire"),
                         )
                         .arg(
                             Arg::with_name("expiration-ttl")
+                            .help("Number of seconds for which the entries should be visible before they expire. At least 60")
```

Comment:
should we include the fact that this takes precedence over `expiration`?

Reply:
Yes!